lesson3.js

```diff
@@ -0,0 +1,15 @@
+const ul = document.getElementById('js-list');
+for(let i=1; i<=2; i++) {
+    const li = document.createElement('li');
+    const a = document.createElement('a');
+    a.textContent = `a${i}`;
+    a.href= `a${i}.html`;
+    const img = document.createElement('img');
+    const imgbox = ['/img/bookmark.png','/img/message.png'];
+    img.src = imgbox[i-1];
+    const fragment = document.createDocumentFragment();
+    {
+      fragment.appendChild(li).appendChild(a).insertBefore(img, a.firstChild);
+    }
```

Comment:
[IMO]
You don't need to nest them with { } here.

Reply:
As you pointed out, it's easier to see if it's not nested. Thanks for the advice!